### PR TITLE
Clarify the meaning of high/low in logits and labels

### PR DESCRIPTION
**Pull request by shwang**
Thanks to @qxcv for pointing out that our placeholders were confusing, especially since the meaning of high/low logits is different between the GAIL and AIRL paper.

I ended up making generator predictions correspond with label=1 and high logits.

In AIRL we talked about using flipped logits and labels (expert predictions correspond with label=1) but this didn't seem necessary even for clarity because we don't end up using these quantities to construct the reward (see the removed part of https://github.com/HumanCompatibleAI/imitation/commit/fcab4531976cb27fd50aa4698497371dab1147a1).

Let me know if it seems better the other way around.

### CODE REVIEW

**Comment by shwang** [comment]
Going to try to make some before and after plots for this too, just in case.

**Comment by shwang** [comment]
Before:  https://www.notion.so/chaiberkeley/Before-PRs-156-161-2c7970b81de242e8a8859169245c2782
After: https://www.notion.so/chaiberkeley/After-PR-156-refactor-AIRL-labels_ph-20392876c3724ed49a87de4ece7293d3

**Comment by shwang** [comment]
Tensorboard plots before: https://www.notion.so/chaiberkeley/Before-PRs-156-161-2c7970b81de242e8a8859169245c2782
Tensorboard plots after: https://www.notion.so/chaiberkeley/After-PR-161-refactor-AIRL-labels_ph-4ad9cb4ceb654867b0a1742f8fd37b82
